packages/expo-cli/package.json

```diff
@@ -90,6 +90,7 @@
     "enquirer": "^2.3.2",
     "envinfo": "5.10.0",
     "es6-error": "3.2.0",
+    "expo-optimize": "0.0.0",
```

Comment:
Shouldn't we remove the dependency on `expo-optimize` from the default Expo CLI installation if we intend this to help with installation issues related to `@expo/image-utilts` / `sharp-cli`? Otherwise `npm i -g expo-cli` will still try to pull in Sharp.

Reply:
it's part of the deprecation. Running `expo optimize` will warn you to switch over for a little while.

Reply:
Sounds good to me